twine/utils.py

```diff
@@ -295,3 +298,28 @@ def bool_from_env(val: Optional[str]) -> bool:
         """Allow '0' and 'false' and 'no' to be False."""
         falsey = {"0", "false", "no"}
         return bool(val and val.lower() not in falsey)
+
+
+_MAX_VERBOSITY = 5
+
+_VERBOSITY_TO_LOG_LEVEL = {
+    0: logging.WARNING,
+    1: logging.INFO,
+    2: logging.DEBUG,
+    3: _MAX_VERBOSITY,
+}
+
+
+def setup_logging(args_verbosity: int) -> None:
+    """Set up the logger and logging based on the verbosity inputted by the user."""
+    # 3 Vs are the maximum verbosity
+    if args_verbosity > 3:
+        args_verbosity = 3
+
+    log_level = _VERBOSITY_TO_LOG_LEVEL[args_verbosity]
+    logger = logging.getLogger("LOGGER")
+    handler = logging.StreamHandler(sys.stdout)
+    handler.setLevel(log_level)
+    logger.addHandler(handler)
+    # Setting a level of verbosity for the logger
+    logger.setLevel(log_level)
```

Comment:
I *think* we only need one `setLevel` call, and the method name is sufficiently descriptive to not need a comment:

```suggestion
    logger.addHandler(handler)
    logger.setLevel(log_level)
```

Reply:
I'm guessing this is cribbed liberally from Flake8 and I do think that I needed to call to `setLevel` but that may have been a 2/3 compatibility issue. We should test which combination of set levels is actually appropriate rather than guessing